rele/config.py

```diff
@@ -103,9 +103,11 @@ def load_subscriptions_from_paths(sub_module_paths, sub_prefix=None, filter_by=N
                 subscription.set_filters(filter_by)
 
             if subscription.name in subscriptions:
+                found_subscription = subscriptions[subscription.name]
                 raise RuntimeError(
                     f"Duplicated subscription found: {subscription.name}. Handled by "
```

Comment:
I would reformulate this to:
`Duplicate subscription name found: {subscription.name]. subs {sub1} and {sub2} collide`